tasks/files/snapshot.py

```diff
@@ -213,7 +263,7 @@ def get_snapshot_name(lv_name, prefix, suffix):
     else:
         suffix_str = ""
 
-    return prefix_str + lv_name + suffix_str
+    return prefix_str + lv_name + "_" + suffix_str
```

Comment:
This means the snapshot name will always have a `"_"` in it, even if `suffix_str` is `""`?  So you could have a name ending with `_`?

Reply:
yes, I was thinking the _ would be a delimiter between the LV name and the snapset name.  I added a check to make sure suffix_str of "" is not allowed.